waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -607,16 +611,19 @@ proc addAll*(rlnInstance: RLN[Bn256], list: seq[IDCommitment]): bool =
 type RegistrationEventHandler  = proc(pubkey: Uint256, index: Uint256): void {.gcsafe, closure, raises: [Defect].}
 
 
-proc subscribeToGroupEvents(ethClientUri: string, contractAddress: Address, blockNumber: string = "0x0", handler: RegistrationEventHandler) {.async, gcsafe.} = 
+proc subscribeToGroupEvents(ethClientUri: string, ethAccountAddress: Address, contractAddress: Address, blockNumber: string = "0x0", handler: RegistrationEventHandler) {.async, gcsafe.} = 
   ## connects to the eth client whose URI is supplied as `ethClientUri`
   ## subscribes to the `MemberRegistered` event emitted from the `MembershipContract` which is available on the supplied `contractAddress`
   ## it collects all the events starting from the given `blockNumber`
   ## for every received event, it calls the `handler`
   
   # connect to the eth client
-  let web3 = await newWeb3(ETH_CLIENT)
```

Comment:
If `ETH_CLIENT` is no more needed (replaced by input `ethClientUri`) then probably you can remove https://github.com/status-im/nwaku/blob/993781a137a6ce63d3be4c7cd2d9e8bd5b403c98/waku/v2/protocol/waku_rln_relay/waku_rln_relay_types.nim#L96-L98

Reply:
It is used for the testing purposes.